omnigibson/utils/python_utils.py

```diff
@@ -277,18 +278,21 @@ def create_class_from_registry_and_config(cls_name, cls_registry, cfg, cls_type_
     return cls(**cls_kwargs)
 
 
-def get_uuid(name, n_digits=8):
+def get_uuid(name, n_digits=8, deterministic=True):
```

Comment:
maybe we should just allow `deterministic=True`? I can't really think of a case where we want `deterministic=False`.